src/main/java/uk/gov/justice/digital/service/ValidationService.java

```diff
@@ -51,7 +51,7 @@ public Dataset<Row> handleValidation(SparkSession spark, Dataset<Row> dataFrame,
 
     @VisibleForTesting
     Dataset<Row> validateRows(Dataset<Row> df, SourceReference sourceReference, StructType inferredSchema) {
-        StructType schema = withMetadataFields(sourceReference.getSchema());
+        StructType schema = withScdFields(withMetadataFields(sourceReference.getSchema()));
```

Comment:
Adds the columns `checkpoint_col` and `update_type` to the in-memory schema to allow successful schema validation. This is required because the raw data being validated now contains both columns